src/components/Field/FieldTypes.tsx

```diff
@@ -40,6 +40,10 @@ export interface FieldDef<T = any, U = any> {
 	 * Significant name related to its field.
 	 */
 	name: string;
+	/**
+	 * ChipList reference to get its height

```

Comment:
This ref is not specific for only the ChipList. Remember that FieldDef affects the "definition" of all fields. Please update this text to say something like "React ref for the field"